configs/bootstrap.fragment

```diff
@@ -0,0 +1,6 @@
+BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL)/board/adi/ev-sc598-som/overlay"
+BR2_TARGET_ROOTFS_CPIO=y
+BR2_PACKAGE_MTD=y
+BR2_PACKAGE_MTD_FLASHCP=y
+BR2_SYSTEM_BIN_SH_BUSYBOX=y
+BR2_ROOTFS_POST_BUILD_SCRIPT="$(BR2_EXTERNAL)/board/adi/ev-sc598-som/post-build.sh"
```

Comment:
We'll also want to disable all the extra options in the mainline defconfig we don't need to reduce the size of the image.